core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/README.md

```diff
@@ -20,13 +20,18 @@ Use the java command to run the created benchmark class with the following suppo
 #### Running the Benchmark class directly
 ```shell
 java -cp $CLASSPATH \
-com.nvidia.spark.rapids.tool.benchmarks.SingleThreadedQualToolBenchmark \
+org.apache.spark.rapids.tool.benchmarks.SingleThreadedQualToolBenchmark \
 -i 3 -w 3 -a " --output-directory output eventlogs"
 ```
+`CLASSPATH` should include the path relative to the Benchmarking class being passed. 
+Refer to the example below for the same.
```

Comment:
Which example is this referring to? I think the below example is for a separate case.

Reply:
Referring to the command below where we are passing $RAPIDS_TOOLS_JAR. Mainly want to highlight that the cp should be set such that the Benchmarking class name should include the relative path to the cp -`org.apache.spark.rapids.tools.benchmarks.SingleThreadedQualToolBenchmark` in this case containing the package structure information

Reply:
Updated the wording to make this clear